pkg/storaged/block/format-dialog.jsx

```diff
@@ -242,6 +247,10 @@ function format_dialog_internal(client, path, start, size, enable_dos_extended,
     if (old_opts == undefined)
         old_opts = initial_mount_options(client, block);
 
+    old_dir = client.strip_mount_point_prefix(old_dir);
+    if (old_dir === false)
+        return Promise.reject(_("This device can not be used for the installation target."));
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19352-20240102-111247-0a02f8c6-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)